cfme/tests/test_rest.py

```diff
@@ -26,7 +26,7 @@
     scope="module"
 )
 
-pytestmark = [pytest.mark.ignore_stream("5.2")]
+pytestmark = [pytest.mark.ignore_stream("5.3")]
```

Comment:
yeh I'm good with that.....even the bit at the top to ignore 5.3 could be removed really